### PR TITLE
Fix #1561: Small to medium default text size

### LINKED ISSUE #1561: The default text size should be MEDIUM_TEXT_SIZE instead of SMALL_TEXT_SIZE
While creating the admin profile in `ProfileActivityPresenter` and while adding new profile using `AddProfileActivity' we should set text size as `MEDIUM_TEXT_SIZE` instead of `SMALL_TEXT_SIZE`.

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes #1561 

I have updated the implementation such that the `addProfile` function in domain layer should automatically set the default values for `StoryTextSize`, `AudioLanguage` and `AppLanguage`. This change is done keeping in mind the current mocks.

Now, after making that change, I found one mistake in app layer test cases which is, somewhere in PRs we added one more `addProfile` item in [ProfileTestHelper](https://github.com/oppia/oppia-android/blob/ce6c082df80234b12e1b980c2ab305a789592bea/domain/src/main/java/org/oppia/domain/profile/ProfileTestHelper.kt#L39) and because of which the indexes in app layer test cases should have changed but we missed it. So, this PR fixes those indexes too.

**Note for reviewer**: If a particular test case fails, just re-run that single test cases and it will pass.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/app/profile/ProfileActivityPresenter.kt [hunk 1]
```diff
@@ -26,7 +26,7 @@ class ProfileActivityPresenter @Inject constructor(
       allowDownloadAccess = true,
       colorRgb = -10710042,
       isAdmin = true,
-      storyTextSize = StoryTextSize.SMALL_TEXT_SIZE,
+      storyTextSize = StoryTextSize.MEDIUM_TEXT_SIZE,
```
Comment (BenHenning) [inline app/src/main/java/org/oppia/app/profile/ProfileActivityPresenter.kt]: Is the default for regular new profiles correct? Are there tests guarding this?

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks for the quick fix! Just one question.

File: app/src/main/java/org/oppia/app/profile/ProfileActivityPresenter.kt (hunk 1 above)
Comment (rt4914) [reply app/src/main/java/org/oppia/app/profile/ProfileActivityPresenter.kt]: @BenHenning I have updated the description. PTAL.

**Review by BenHenning — APPROVED** [review]
Thanks @rt4914!

**Pull request merged**